app/routes.py

```
@@ -575,13 +733,13 @@ async def generate_persona_image(request: PersonaImageRequest) -> PersonaImageRe
             name=name, prompt=prompt, style="realistic", size=(512, 512)
         )
 
-        if not image_generation_result: # Check if the string is empty/None
+        if not image_generation_result:  # Check if the string is empty/None
             raise HTTPException(
                 status_code=status.HTTP_400_BAD_REQUEST,
-                detail="Failed to generate image: No URL returned."
+                detail="Failed to generate image: No URL returned.",
             )
```

Ruff (0.15.21)

[warning] Abstract `raise` to an inner function

(TRY301)